tests/ppf_test.go

```diff
@@ -0,0 +1,117 @@
+}
+
+// Test to check if the ParseTimeTable function returns an error when the file does not exist or the file type is invalid
+func TestParseTimeTableInvalidFile(t *testing.T) {
+	_, errFilePath := ppf.ParseTimeTable("data/actor/invalid-file-path.csv")
+	assert.NotNil(t, errFilePath)
+
+	_, errFileType := ppf.ParseTimeTable("data/actor/invalid-file-type.txt")
+	assert.NotNil(t, errFileType)
+}
+
+// Test to check if the ParseTimeTable function returns an error when the header of the file is invalid
+func TestParseTimeTableInvalidHeader(t *testing.T) {
+	_, err := ppf.ParseTimeTable("data/actor/invalid-header.csv")
+	assert.NotNil(t, err)
+}
+
+// Test to check if the ParseTimeTable function returns an error when the time availability is invalid
+func TestParseTimeTableInvalidAvailability(t *testing.T) {
+	_, err := ppf.ParseTimeTable("data/actor/invalid-times.csv")
+	assert.NotNil(t, err)
+}
+
+// Test to check if ParseTechnicalGuide	function returns the correct data for a valid technical guide
+func TestParseTechnicalGuide(t *testing.T) {
+	// Create the Member list
+	actor := ppf.NewMember("actor", ppf.Actor, [][]time.Time{{time.Now(), time.Now()}})
+	actor2 := ppf.NewMember("actor2", ppf.Actor, [][]time.Time{{time.Now(), time.Now()}})
+	crew := ppf.NewMember("crew", ppf.Crew, [][]time.Time{{time.Now(), time.Now()}})
+
+	// Create times (10 minutes)
+	tenMinutes := time.Duration(600000000000)
+
+	// Test for a valid technical guide
+	shots, err := ppf.ParseTechnicalGuide("data/valid-tech-guide.csv", []ppf.Member{actor, actor2, crew})
+	// Basic checks
+	assert.Nil(t, err)
+	assert.Equal(t, 2, len(shots))
+	// Check the first shot
+	firstShot := shots[0]
+	assert.Equal(t, tenMinutes, firstShot.EstimatedDuration)
+	assert.ElementsMatch(t, []ppf.Member{actor}, firstShot.Cast)
+	assert.ElementsMatch(t, []ppf.Member{crew}, firstShot.Crew)
+	assert.Equal(t, "Test Location", firstShot.Setting)
+	assert.Equal(t, "INTRO", firstShot.Scene)
+	assert.Equal(t, "P1", firstShot.Id)
+	assert.Equal(t, "Test Details", firstShot.Details)
+}
+
+// Test to check if ParseTechnicalGuide function returns an error when the file does not exist or the file type is invalid
+func TestParseTechnicalGuideInvalidFile(t *testing.T) {
+	_, errFilePath := ppf.ParseTechnicalGuide("data/invalid-tech-guide.csv", []ppf.Member{})
+	assert.NotNil(t, errFilePath)
+
+	_, errFileType := ppf.ParseTechnicalGuide("data/invalid-tech-guide.txt", []ppf.Member{})
+	assert.NotNil(t, errFileType)
+}
+
+// Test to check if ParseTechnicalGuide function returns an error when the header of the file is invalid
+func TestParseTechnicalGuideInvalidHeader(t *testing.T) {
+	_, err := ppf.ParseTechnicalGuide("data/invalid-tech-guide-header.csv", []ppf.Member{})
+	assert.NotNil(t, err)
+}
+
+// Test to check if ParseTechnicalGuide function returns an error when the cast or crew of a shot is invalid
+func TestParseTechnicalGuideInvalidCastCrew(t *testing.T) {
```

Comment:
¿En cuál de estos tests estás comprobando si Aquiles puede grabar en la misma localización la mayor cantidad de tiempo posible? (#6)

Reply:
Esto no debería testearse en esta milestone, porque no estoy aún creando el plan de rodaje como tal. He movido esa HU a la siguiente milestone.

Reply:
Ese es el HU al que se refiere el issue que has indicado en el código escrito. Si te vas a centrar en otra (aparentemente, parsear los planes de rodaje) lo conveniente sería que hicieras una HU específica para eso, porque no veo cómo se puede encajar fácilmente en ninguna de [las existentes](https://github.com/diagmatrix/maybe-better-maybe-worse/issues?q=is%3Aissue+is%3Aopen+label%3Auser-stories)
Por otro lado, si no vas a trabajar con esa HU, no vas a trabajar con esse issue, y todo el código que hayas generado usándolo quedaría fuera de este PR, ¿no?

Reply:
Había pensado en #3 , ya que al final parsear un plan de rodaje puede hacerse si se modifican los planos